src/log.js

```diff
@@ -57,10 +61,8 @@ class Log extends GSet {
     this._storage = ipfs
     this._id = id || randomId()
 
-    // Signing related setup
-    this._keystore = this._storage.keystore
-    this._key = key 
-    this._keys = Array.isArray(keys) ? keys : [keys]
+    // Access Controller
+    this._acl = acl
```

Comment:
Like @fazo96 mentioned in his comment (https://github.com/orbitdb/ipfs-log/pull/128#issuecomment-399921075), maybe we should add some validation here that checks if the ACL exists and if it follows the "interface" we need. If it doesn't, we throw an error. Is there a scenario where it wouldn't be necessary?